client/src/components/Support Page/Tabs/PageOne/SoftwareProblems.js

```diff
@@ -0,0 +1,92 @@
+import React from 'react';
+import { Grid } from '@material-ui/core';
+import { UseForm, Form } from '../../../UI Elements/UseForm';
+import { Input, MultiInput } from '../../../UI Elements/Input'
+import Button from '../../../UI Elements/Button'
+
+const initialFValues = {
+    mobile: '',
+    whatsapp: '',
+    email: '',
+    query: ''
+}
+
+
+
+const SoftwareProblems = () => {
+    const { 
+        values, 
+        setValues, 
+        handleInputChange,
+        error,
+        setError } = UseForm(initialFValues);
+
+    const validate = () => {
+        let temp = {}
+        temp.mobile = values.mobile.length>9?"":"Enter a valid number"
+        temp.whatsapp = values.whatsapp.length>9?"":"Enter a valid number"
+        temp.email = (/$^|.*@.*..*/).test(values.email)?"":"Email is not valid"
+        temp.query = values.query?"":"This field is required"
+        setError({
+            ...temp // What is this?
```

Comment:
This is object destructuring. Read more [here](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Destructuring_assignment).

Please remove the comment once you are doing reading this.